extensions/amp-analytics/0.1/variables.js

```diff
@@ -309,73 +309,63 @@ export class VariableService {
       }
 
       let value = options.getVar(name);
+      const urlReplacements = Services.urlReplacementsForDoc(element);
```

Comment:
@zhouyx  Do you think we should just add `urlReplacements` to the function parameters (since we're always forcing element to be passed in anyway)? 

Could save us work in terms of having to get the `urlReplacementsForDoc(element)` at every level of expansion (granted, that work might be very minimal).

Reply:
Sorry about the delay in response. `urlReplacementsForDoc(element)` is simply retrieving item from an object. I'd definitely prefer to only pass `element`. Because cost wise it's the same, but code wise it's cleaner.

Reply:
Ok, sgtm!